crates/common/logged_command/Cargo.toml

```diff
@@ -9,9 +9,12 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
+camino = { workspace = true }
 log = { workspace = true }
 nix = { workspace = true }
 shell-words = { workspace = true }
+tedge_api = { workspace = true }
```

Comment:
Having a dependency of `common/logged_command` on `tedge_api` is wrong being a dependency from a low-level module on a high-level API.

- Is this dependency really required? 
- Alternatively, this `logged_command` module could be moved under the `tedge_api` crate. After all, this would be simply acting the fact `CommandLog` are really workflow specific.

Reply:
Resolved in https://github.com/thin-edge/thin-edge.io/pull/2895/commits/5f76e273db7b27d3358f1bb04158db789a35ff69